src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    private static final Task[] tasks = new Task[100];
+
+    public static void command(String line) {
+        String exitCommand = "bye";
+        String toDo = "list";
+        String checkAsDone = "mark";
+        int numberOfTasks = 0;
+
+        String[] tokens = line.split(" ", 2);
+        String command = tokens[0];
+
+        while (!command.equals(exitCommand)) {
```

Comment:
Currently, this entire method is too long. You can try to keep a method short by extracting its content out to other methods whenever appropriate.